### PR TITLE
Fix: Time Series panel crashes when setting a max value with no min value

### LINKED ISSUE #13583: Time Series panel crashes when setting a max value with no min value
### Describe the Bug

When setting a max value while leaving the min value automatic the minimum value is seen as higher than the max. This causes the page to crash.

### To Reproduce

1. Create a time series
2. Set the max value
3. Leave the minimum value unset (Automatic)

### Errors Shown

axis.min cannot be greater than axis.max

### What version of Directus are you using?

main ([4e6c361](https://github.com/directus/directus/commit/4e6c361a0e6cfabb9b71bedecc7a54ccf6fe8bf0))

### What version of Node.js are you using?

v16.13.1

### What database are you using?

postgres

### What browser are you using?

Chromium

### How are you deploying Directus?

running locally

### PR DESCRIPTION
**Pull request by jamescammarano**
## Description

Previously when working with the time series panel if you set a max y-axis value but did not specify a minimum y-axis value the whole dashboard would crash with the error:

> axis.min cannot be greater than axis.max

Now, the app will automatically use 0 for the minimum when a maximum value is selected. This still allows an undefined value for both when a user wants to have ApexCharts decide the y-axis range. 

The app will also automatically flip the min and max when the minimum is above the maximum. 

I made a computed object so that a refresh is triggered when `props.min` or `props.max` values change.

Fixes #13583

## Type of Change

- [x] Bugfix
- [ ] New Feature
- [ ] Refactor / codestyle updates
- [ ] Other, please describe:

## Requirements Checklist

- [ ] New / updated tests are included
- [x] All tests are passing locally
- [x] Performed a self-review of the submitted code

If adding a new feature:

- [ ] Documentation was added/updated

### CODE REVIEW

**Review by rijkvanzanten — APPROVED** [review]

File: app/src/panels/time-series/time-series.vue [hunk 1]
```diff
@@ -107,6 +107,21 @@ export default defineComponent({
 		const chartEl = ref();
 		const chart = ref<ApexCharts>();
 
+		const yAxisRange = computed(() => {
+			let min = isNil(props.min) ? undefined : Number(props.min);
+			let max = isNil(props.max) ? undefined : Number(props.max);
+
+			if (max && !min) {
```
Comment (rijkvanzanten) [inline app/src/panels/time-series/time-series.vue:114]: Actually, we should do a more explicit "is undefined" check here. the number `0` could be a valid option for the value, but is also considered falsy in JS, so would trigger these if statements wrong

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

File: app/src/panels/time-series/time-series.vue [hunk 2]
```diff
@@ -107,6 +107,21 @@ export default defineComponent({
...
+				min = 0;
+			}
+
+			if (max && min && max < min) {
+				max = min;
+				min = Number(props.max);
+			}
+			return { max, min };
```
Comment (rijkvanzanten) [inline app/src/panels/time-series/time-series.vue:122]: And lets add a return in between here for code style beauty reasons 👨🏻‍🎨

Comment (jaycammarano) [reply app/src/panels/time-series/time-series.vue:122]: Where?

Comment (jaycammarano) [reply app/src/panels/time-series/time-series.vue:122]: Like line break?

Comment (rijkvanzanten) [reply app/src/panels/time-series/time-series.vue:122]: Yup!

```suggestion
			}

			return { max, min };
```

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**